hs628_tools/random-forest.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import pandas as pd
+from sklearn.model_selection import train_test_split
+from sklearn.ensemble import RandomForestClassifier, RandomForestRegressor
+
+def rf(features, target, model='classifier'):
```

Comment:
As I understand your intention, this rf function is meant to be a convenience function for accessing RandomForestClassifier and RandomForestRegressor; it would be more convenient if the user also had the option to provide non-default values for their various respective parameters, if they chose.  You can provide that flexibility by including ```**kwargs``` in the specified parameter list. 
```python
def rf(features, target, model='classifier', **kwargs):
    train_features, train_target, test_features, test_target = train_test_split(features, target, test_size=0.25, random_state=13)
    if model == 'classifier':
        my_defaults = {'n_estimators': 10, 'criterion': 'gini', 'max_features': 'auto', 'n_jobs': 1, 
                                 'random_state': 13, 'class_weight': 'balanced'}
        for k, v in my_defaults.iteritems():
            if k not in kwargs:
            kwargs[k] = v
        clf = RandomForestClassifier(**kwargs)
```

I find the [standard docs](https://docs.python.org/2/tutorial/controlflow.html#keyword-arguments) on this to not be great.  Here is [one reference](https://stackoverflow.com/a/1769452) and [another](http://treyhunner.com/2018/04/keyword-arguments-in-python/) that might help more, and of course you can google lots of others.

Reply:
I see, the stackoverflow link explained it very well. So kwargs provides a user-defined dictionary of value to pass into 
the guts of a function? Should I provide docstrings that explain the formatting of kwargs, such as required key-value pairs?

Reply:
Resolved, please see commit below.